src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -24,14 +24,13 @@
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + PREFIX_USERTYPE + "USERTYPE...\n"
```

Comment:
Should there still be trailing dots since only 1 user type should be used in the command?

Reply:
Ah yes, will edit that part! Could be confusing if users might still perceive User Type to behave similarly as Tag